terraform/services/main.tf

```diff
@@ -33,8 +17,8 @@ resource "incus_profile" "profile" {
   description = "Profile to be used by the service containers"
 
   config = {
-    "limits.cpu"    = 1
-    "limits.memory" = "1GiB"
+    "limits.cpu"       = 1
```

Comment:
We probably should do strings for all of those config keys even if ints are accepted.
(not something you introduced, just something I noticed now)